### PR TITLE
AC-627: Change ImageView color depending upon theme

### PR DESCRIPTION
**Pull request by deepak140596**
## Description of what I changed

Changed Tint of ImageViews in the Dashboard buttons depending on the theme set.

## Issue I worked on

JIRA Issue: https://issues.openmrs.org/browse/AC-627

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

- [ ] I have **added tests** to cover my changes. (If you refactored
existing code that was well tested you do not have to add tests)

- [x] All new and existing **tests passed**.

- [x] My pull request is **based on the latest changes** of the master branch.

### CODE REVIEW

**Comment by deepak140596** [comment]
<p float="left">
  
</p>

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/dashboard/DashboardFragment.java [hunk 1]
```diff
@@ -333,4 +340,14 @@ public void onClick(View v) {
                 break;
         }
     }
+
+    private void changeColorOfImageView() {
+        int colorResId = R.color.green;
```
Comment (f4ww4z) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/dashboard/DashboardFragment.java]: Not a good enough variable name. Also make this a final class variable instead.

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/dashboard/DashboardFragment.java [hunk 2]
```diff
@@ -333,4 +340,14 @@ public void onClick(View v) {
                 break;
         }
     }
+
+    private void changeColorOfImageView() {
```
Comment (f4ww4z) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/dashboard/DashboardFragment.java]:
```suggestion
    private void changeColorOfDashboardIcons() {
```

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/dashboard/DashboardFragment.java (hunk 1 above)
Comment (deepak140596) [reply openmrs-client/src/main/java/org/openmrs/mobile/activities/dashboard/DashboardFragment.java]: @f4ww4z , changed the name to `greenColorResId` .

**Comment by deepak140596** [comment]
@f4ww4z , Please review this PR

**Pull request merged**